version: '3.3'

services:
  pye36:
    build:
      context: .
      dockerfile: ./docker/Dockerfile
      args:
        PY_VER: 3.6
    image: falldog/pyconcrete-tester:3.6

  pye37:
    build:
      context: .
      dockerfile: ./docker/Dockerfile
      args:
        PY_VER: 3.7
    image: falldog/pyconcrete-tester:3.7

  pye38:
    build:
      context: .
      dockerfile: ./docker/Dockerfile
      args:
        PY_VER: 3.8
    image: falldog/pyconcrete-tester:3.8

  pye39:
    build:
      context: .
      dockerfile: ./docker/Dockerfile
      args:
        PY_VER: 3.9
    image: falldog/pyconcrete-tester:3.9

  pye310:
    build:
      context: .
      dockerfile: ./docker/Dockerfile
      args:
        PY_VER: "3.10"
    image: falldog/pyconcrete-tester:3.10
